.docker/install-plugin-theme.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install plugins
+echo "Installing plugins"
+wp --allow-root --url=wordcamp.test plugin install akismet buddypress bbpress jetpack wp-multibyte-patch wordpress-importer polldaddy liveblog wp-super-cache custom-content-width camptix-network-tools email-post-changes tagregator supportflow camptix-pagseguro camptix-payfast-gateway camptix-trustpay camptix-trustcard camptix-mercadopago camptix-kdcpay-gateway campt-indian-payment-gateway camptix
+
+# Install themes
+echo "Installing themes"
+wp --allow-root --url=wordcamp.test theme install twentyten twentyeleven twentytwelve twentythirteen twentyfourteen
```

Comment:
`MUST` - Also `p2`

Reply:
Same as https://github.com/WordPress/wordcamp.org/pull/113#discussion_r281663564

Reply:
Added themes in https://github.com/WordPress/wordcamp.org/pull/113/commits/db214d69ebe62cfa4be3af4e8432092c90f72061